src/components/Header/Header.tsx

```diff
@@ -36,12 +36,13 @@ const Header = ({
   const handleToggle = () => {
     if (navbarStatus === true) {
       changeNavbarStatus(false);
```

Comment:
i know this is old but this should be `isNavbarOpen`  and `setIsNavbarOpen`. You want to reflect the variable type in the naming. For instance, `navbarStatus` would assume it could be any string value such as "open", "closed", etc not true or false.